lmp/host_tests.py

```diff
@@ -114,7 +114,6 @@ def _test_doanac_410c():
 DEVICES = {
     # ota+/jobserv name:  (POLIS_ID, test_function)
     'doanac-minnowboard': (DOANAC, _test_doanac_intel),
-    'doanac-dragonboard-410c': (DOANAC, _test_doanac_410c),
```

Comment:
can you also delete the `_test_doanac_410c` function in that file?

Reply:
Sure, forgot about that one, updated & pushed.